src/routes/b/[room]/+page.svelte

```diff
@@ -208,7 +212,7 @@
 	{#if sent}
 		<span class="text-xl font-light">✨ Sent your message </span>
 	{:else if profanityWarning}
-		<span class="text-error"> ⚠️ Message has not been sent because it contains profanity </span>
+		<span style="color: #c95050;">⚠️ Message has not been sent because it has profanity words</span>
```

Comment:
... why ?

Reply:
contains profanity is more like . .. grammatically correct

Reply:
U're right I was planning changing it to "the host doesn't allow profanity words"

Reply:
⚠️ Message has not been sent because the room host doesn't allow profanity words

Reply:
Yeah that makes a lot more sense , thanks lol